ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePageVulnerabilities.tsx

```diff
@@ -121,13 +122,13 @@ function ImagePageVulnerabilities({
     pagination,
 }: ImagePageVulnerabilitiesProps) {
     const { isFeatureFlagEnabled } = useFeatureFlags();
-    const isUnifiedDeferralsEnabled = isFeatureFlagEnabled('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL');
```

Comment:
The `useHasRequestExceptionsAbility()` hook encapsulates both this feature flag check as well as the permission check.